src/check/package-json.test.js

```diff
@@ -28,7 +25,7 @@ tap.ok(
   'package.json has "author" property'
 );
 
-tap.equal(pkg.main, 'es5/plugin.js', 'package.json "main" is "es5/plugin.js"');
+tap.equal(pkg.main, 'dist/es5/index.js', 'package.json "main" is "dist/es5/index.js"');
```

Comment:
we also need to check for:
* `jsnext:main` here, as it is required by spellbook
* `generator-videojs-plugin.version` 
* check the files array (to make sure everything is published, but only what should be published.)
* is there any way to check the `.yo.rc` to verify that the `spellbook` configuration in `package.json` matches here?
* such as ie8, and css off
* should we check ghooks here against the `.yo-rc`